src/scenes/StageManager.ts

```diff
@@ -57,8 +55,15 @@ export default class StageManager{
 
     /** Map of Scenes by Scene IDs */
     private scenes: {[key:string]:typeof Scene} = {};
+    public get scale():number{
+        console.warn('scale is deprecated, please reference viewFrame for stage size info');
```

Comment:
I would argue that the code is obsolete (ie no longer working, at least not correctly since it will always return 1). https://stackoverflow.com/questions/9208091/the-difference-between-deprecated-depreciated-and-obsolete